chess/src/game.rs

```diff
@@ -13,6 +13,8 @@ use crate::timer::Timer;
 use std::collections::HashSet;
 use web_time::Duration;
 
+const MAX_FEN_STR: usize = 87;
```

Comment:
you had `80` but now it's `87`. Is 87 correct?

Reply:
Yeah before I just gave a rough number, but I looked it up and I guess 87 is the longest possible FEN string